modules/rostests/apitests/crt/ceil.c

```diff
@@ -0,0 +1,109 @@
+/*
+* PROJECT:     ReactOS api tests
+* LICENSE:     MIT (https://spdx.org/licenses/MIT)
+* PURPOSE:     Tests for ceil / ceilf
+* COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
+*/
+
+#define _CRTBLD // we don't want inline ceilf!
+#include "math_helpers.h"
+
+#ifdef _MSC_VER
+#pragma function(ceil)
+// ceilf is not available as an intrinsuc
+#endif
+
+static TESTENTRY_DBL s_ceil_tests[] =
+{
+    /* Special values */
+    { 0x0000000000000000 /* 0.000000000000000e+000 */, 0x0000000000000000 /* 0.000000000000000e+000 */ },
```

Comment:
Suggestion for readability (in these comments): align the digits vertically, and the signs too, like:
```
/*  0.000000000000000e+000 */ ...
/* -0.000000000000000e+000 */ ...
etc.
```

Reply:
Fixed